packages/core/index.d.ts

```diff
@@ -51,6 +51,8 @@ declare module '@rjsf/core' {
         uiSchema?: UiSchema;
         validate?: (formData: T, errors: FormValidation) => FormValidation;
         widgets?: { [name: string]: Widget };
+        // WARNING: This exists for internal react-jsonschema-form purposes only. Use ONLY if you know what you are doing
```

Comment:
I think we should use `jsDoc` here. 

```suggestion
       /** WARNING: This exists for internal react-jsonschema-form purposes only. Use ONLY if you know what you are doing */
```